docs/site/Server.md

```diff
@@ -338,6 +338,51 @@ export class HelloWorldApp extends Application {
 You can also add multiple servers in the constructor of your application class
 as shown [here](Application.md#servers).
 
+### Enhance OpenAPI Specification
+
+The rest server exposes a function `getApiSpec()` to retrieve its OpenAPI
+specifications:
+
+```ts
+// in code, retrieve the OpenAPI spec by `getApiSpec()`
+const spec = await app.restServer.getApiSpec();
+```
+
+An application's OAS is mainly generated from
+[controllers](https://loopback.io/doc/en/lb4/Controllers.html) and their
+members' metadata. Besides this, we would also like to contribute specifications
+from other places. Therefore we introduced
+[OpenAPI specification enhancer](Extending-OpenAPI-specification.md) to
+customize it.
+
+You can read the page
+[Extending OpenAPI specification](Extending-OpenAPI-specification.md) to get
+familiar with its concepts and usages.
+
+The REST server has a built-in enhancer service to scan all the enhancers bond
```

Comment:
Not sure if it should be `bounded` or `bind`. You might also want to look at other instances too. Thanks.

```suggestion
The REST server has a built-in enhancer service to scan all the enhancers bounded
```